kubernetes/resource_kubernetes_job_v1_test.go

```diff
@@ -516,3 +592,53 @@ func testAccKubernetesJobV1Config_modified(name, imageName string) string {
   wait_for_completion = false
 }`, name, imageName)
 }
+
+func testAccKubernetesJobV1Config_customizeDiff_ttlZero(name, imageName string) string {
+	return fmt.Sprintf(`
+resource "kubernetes_job_v1" "test" {
+  metadata {
+    name = "%s"
+  }
+  spec {
+    ttl_seconds_after_finished = 0
+    template {
+      metadata {}
+      spec {
+        container {
+          name    = "wait-test"
+          image   = "%s"
+          command = ["sleep", "60"]
+        }
+        restart_policy = "Never"
+      }
+    }
+  }
+  wait_for_completion = false
+}
+`, name, imageName)
+}
+
+func testAccKubernetesJobV1Config_customizeDiff_ttlFive(name, imageName string) string {
```

Comment:
Instead of having 2 configs, you could just make the ttl value of a parameter to the function and template it in.